src/contractIntegration/IPFSManagementInteraction/IPFSManagementInteraction.ts

```diff
@@ -1,3 +1,5 @@
+/*eslint no-unused-vars: "error"*/
```

Comment:
@HectorGerardoZV Why ignore the unused variables error? If there are any unused variables, it is better to remove them for cleaner code.

Reply:
Sorry, it was a mistake, I'm going to fix it.